view/frontend/templates/js/replacejs.phtml

```diff
@@ -1485,6 +1489,15 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
             'jquery',
             'Magento_Checkout/js/model/quote',
             ], function ($, quote) {
+                var prevAddress;
+                //If billing address has been changed we need to recreate the order token.
+                quote.billingAddress.subscribe(function(newAddress){
+                    if((!newAddress ^ !prevAddress || prevAddress.getKey() !== newAddress.getKey())){
```

Comment:
can we save only prevAddress.key?

Reply:
I found a different solution for this while checking whole code fixed the old billingAddress listener